### PR TITLE
Create composer.lock

### PR DESCRIPTION
**Pull request by Derky**

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -0,0 +1,2446 @@
+{
+    "_readme": [
+        "This file locks the dependencies of your project to a known state",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
+        "This file is @generated automatically"
+    ],
+    "content-hash": "face0e3a1d77c64d404fd7aec1f4f1f2",
+    "packages": [
+        {
```
Comment (BjornMulder) [inline composer.lock]: haakje

Comment (BjornMulder) [reply composer.lock]: gifbot:wow

Comment (BjornMulder) [reply composer.lock]: [gifbot:mom]

**Comment by Derky** [comment]
[gifbot:wtf]

**Pull request merged**